projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/release/PersonDied.kt

```diff
@@ -0,0 +1,66 @@
+package uk.gov.justice.digital.hmpps.integrations.delius.release
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.datetime.DeliusDateTimeFormatter
+import uk.gov.justice.digital.hmpps.exception.NotFoundException
+import uk.gov.justice.digital.hmpps.integrations.delius.contact.Contact
+import uk.gov.justice.digital.hmpps.integrations.delius.contact.ContactRepository
+import uk.gov.justice.digital.hmpps.integrations.delius.contact.alert.ContactAlert
+import uk.gov.justice.digital.hmpps.integrations.delius.contact.alert.ContactAlertRepository
+import uk.gov.justice.digital.hmpps.integrations.delius.contact.type.ContactTypeCode
+import uk.gov.justice.digital.hmpps.integrations.delius.contact.type.ContactTypeRepository
+import uk.gov.justice.digital.hmpps.integrations.delius.contact.type.getByCode
+import uk.gov.justice.digital.hmpps.integrations.delius.person.Person
+import uk.gov.justice.digital.hmpps.integrations.delius.person.PersonRepository
+import uk.gov.justice.digital.hmpps.integrations.delius.person.manager.probation.PersonManagerRepository
+import uk.gov.justice.digital.hmpps.integrations.delius.person.manager.probation.getByPersonIdAndActiveIsTrueAndSoftDeletedIsFalse
+import java.time.ZonedDateTime
+
+@Service
+class PersonDied(
+    private val personRepository: PersonRepository,
+    private val personManagerRepository: PersonManagerRepository,
+    private val contactTypeRepository: ContactTypeRepository,
+    private val contactRepository: ContactRepository,
+    private val contactAlertRepository: ContactAlertRepository
+) {
+    fun inCustody(nomsId: String, dateTime: ZonedDateTime) {
+        val people = personRepository.findByNomsNumberAndSoftDeletedIsFalse(nomsId)
+        val person = when (people.size) {
+            1 -> people[0]
+            0 -> throw NotFoundException("Person", "nomsNumber", nomsId)
+            else -> throw IllegalStateException("More than one case with the same Noms Number: $nomsId")
```

Comment:
Currently we ignore these errors for normal messaging, should we be doing the same here?
See https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/871a34ba347c7df4373ec8d0417a189d5dabb0d9/projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/event/EventService.kt#L14-L18